src/emm/services/git_service.py

```diff
@@ -177,7 +185,7 @@ def current_branch(self, directory: Optional[Path] = None) -> str:
 
         :param directory: Directory to execute command at.
         """
-        args = ["branch", "--show-current"]
+        args = ["rev-parse", "--abbrev-ref", "HEAD"]
```

Comment:
`branch --show-current` is not support in git 2.17, which the installed version of git on virtual workstations. So I switched this to an equivalent command that is supported.